src/Azure.Functions.Cli/Diagnostics/LoggingFilterHelper.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Diagnostics;
+using Microsoft.Azure.WebJobs.Script.Eventing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Functions.Cli
+{
+    public class LoggingFilterHelper
+    {
+        private const string DefaultLogLevelKey = "default";
+        private readonly IConfigurationRoot _hostJsonConfig;
+        private readonly IList<IDisposable> _eventSubscriptions = new List<IDisposable>();
+
+        // CI EnvironmentSettings
+        // https://github.com/watson/ci-info/blob/master/index.js#L52-L59
+        public const string Ci = "CI"; // Travis CI, CircleCI, Cirrus CI, Gitlab CI, Appveyor, CodeShip, dsari
+        public const string Ci_Continuous_Integration = "CONTINUOUS_INTEGRATION";  // Travis CI, Cirrus CI
+        public const string Ci_Build_Number = "BUILD_NUMBER";  // Travis CI, Cirrus CI
+        public const string Ci_Run_Id = "RUN_ID"; // TaskCluster, dsari
+
+        public LoggingFilterHelper(IConfigurationRoot hostJsonConfig, bool verboseLogging, bool verboseLoggingArgExists)
```

Comment:
This is a bit of a nit -- but could you do `bool? verboseLogging`? and it'd be null value if the arg wasn't set.